alerts/utilization.libsonnet

```diff
@@ -0,0 +1,37 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'utilization-alert.rules',
+        rules: [
+          {
+            alert: 'CephClusterNearFull',
+            expr: |||
+              (ceph_osd_stat_bytes_used / ceph_osd_stat_bytes) * 100 > 85
```

Comment:
afaik, you should use a `sum`, here, i.e.: `sum(ceph_osd_stat_bytes_used) / sum(ceph_osd_stat_bytes) > 0.85 `. (the same goes for the other rule)

Reply:
Ah yes. Will change the rules accordingly.